Dockerfile

```diff
@@ -24,32 +24,39 @@ RUN \
  echo "deb http://ppa.launchpad.net/jcfp/sab-addons/ubuntu bionic main" >> /etc/apt/sources.list.d/sabnzbd.list && \
  echo "deb-src http://ppa.launchpad.net/jcfp/sab-addons/ubuntu bionic main" >> /etc/apt/sources.list.d/sabnzbd.list && \
  echo "**** install packages ****" && \
- if [ -z ${SABNZBD_VERSION+x} ]; then \
-	SABNZBD="sabnzbdplus"; \
- else \
-	SABNZBD_CLEAN=$(echo "${SABNZBD_VERSION}"| sed 's/..$//') && \
-	SABNZBD="sabnzbdplus=${SABNZBD_CLEAN}"; \
- fi && \
  apt-get update && \
  apt-get install -y \
+	jq \
+	libffi-dev \
+	libssl-dev \
 	p7zip-full \
 	par2-tbb \
 	python3 \
 	python3-pip \
-	${SABNZBD} \
+	python3-six \
 	unrar \
 	unzip && \
- pip3 install --no-cache-dir \
-	apprise \
-	chardet \
-	pynzb \
-	requests \
```

Comment:
The `requests` and `apprise` packages are not listed in our `requirements.txt`, so probably good to still install them seperatlty?

Reply:
I added them back in. I still need to make sure I don't accidentally remove some python dependency packages along with python3-pip at the end of the build

Reply:
added cryptography and distutils as ubuntu packages so they don't get attempted to be removed along with pip